packages/relay/tests/lib/config/hbarSpendingPlanConfigService.spec.ts

```diff
@@ -356,6 +384,11 @@ describe('HbarSpendingPlanConfigService', function () {
         };
 
         it('should populate the database with pre-configured spending plans', async function () {
+          // Clear existing spending plans.  Needed in CI environment where the plans are already populated
+          for (const plan of spendingPlansConfig) {
+            await hbarSpendingPlanRepository.delete(plan.id, emptyRequestDetails);
+          }
```

Comment:
Not really sure why this is needed? 

1. We are initializing a completely new redis server for these tests, so there shouldn't be anything populated in the cache in the beginning of this suite
2. We should be clearing the state using `await cacheService.clear(requestDetails)` after every test, in order not to have interdependency between tests

We are either reusing the same redis server, or we are not properly clearing the cache after every test. Please remove this from here and fix the root problem. 🙏

Reply:
Removed.